core/src/main/java/uk/ac/bbsrc/tgac/miso/core/store/QcTargetStore.java

```diff
@@ -0,0 +1,17 @@
+package uk.ac.bbsrc.tgac.miso.core.store;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QC;
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControlEntity;
+
+public interface QcTargetStore {
+  public QC get(long id) throws IOException;
+
+  public QualityControlEntity getEntity(long id) throws IOException;
+
+  public Collection<? extends QC> listForEntity(long id) throws IOException;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)